server/remote/date_time_test.go

```diff
@@ -8,35 +8,37 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/require"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/serializer"
 )
 
 func TestDateTime_Time(t *testing.T) {
 	for _, tc := range []struct {
-		DateTime
+		serializer.DateTime
 		expectedUTC          string
 		expectedString       string
 		expectedPrettyString string
 	}{
 		{
-			DateTime:             DateTime{"2019-11-19T01:00:00.0000005", ""},
+			DateTime:             serializer.DateTime{DateTime: "2019-11-19T01:00:00.0000005", TimeZone: ""},
 			expectedUTC:          "2019-11-19T01:00:00.0000005Z",
 			expectedString:       "2019-11-19T01:00:00Z",
 			expectedPrettyString: "19 Nov 19 01:00 UTC",
 		},
 		{
-			DateTime:             DateTime{"2019-11-19T01:00:00.9999999", "Asia/Hong_Kong"},
+			DateTime:             serializer.DateTime{DateTime: "2019-11-19T01:00:00.9999999", TimeZone: "Asia/Hong_Kong"},
 			expectedUTC:          "2019-11-18T17:00:00.9999999Z",
 			expectedString:       "2019-11-19T01:00:00+08:00",
 			expectedPrettyString: "19 Nov 19 01:00 HKT",
 		},
 		{
-			DateTime:             DateTime{"2019-11-19T01:00:00.9999999", "China Standard Time"},
+			DateTime:             serializer.DateTime{DateTime: "2019-11-19T01:00:00.9999999", TimeZone: "China Standard Time"},
 			expectedUTC:          "2019-11-18T17:00:00.9999999Z",
 			expectedString:       "2019-11-19T01:00:00+08:00",
 			expectedPrettyString: "19 Nov 19 01:00 CST",
 		},
 		{
-			DateTime:             DateTime{"2019-11-19T14:00:00.0000005", "MST"},
+			DateTime:             serializer.DateTime{DateTime: "2019-11-19T14:00:00.0000005", TimeZone: "MST"},
```

Comment:
Is there a need for writing the key names? Can't we use the format used before?

Reply:
This was giving the warning and this was the quick fix